R/zchunk_L202.an_input.R

```diff
@@ -0,0 +1,646 @@
+      L202.UnlimitedRenewRsrcCurves
+
+    L202.UnlimitedRenewRsrcPrice %>%
+      add_title("Unlimited renewable resource price") %>%
+      add_units("1975$/kg") %>%
+      add_comments("A_agUnlimitedRsrcCurves written to all regions") %>%
+      add_legacy_name("L202.UnlimitedRenewRsrcPrice") %>%
+      add_precursors("aglu/A_agUnlimitedRsrcCurves", "common/GCAM_region_names") ->
+      L202.UnlimitedRenewRsrcPrice
+
+    L202.Supplysector_in %>%
+      add_title("Generic supplysector info for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_supplysector written to all regions") %>%
+      add_legacy_name("L202.Supplysector_in") %>%
+      add_precursors("aglu/A_an_input_supplysector", "common/GCAM_region_names") ->
+      L202.Supplysector_in
+
+    L202.SubsectorAll_in %>%
+      add_title("Generic subsector info for inputs to animal production technologies") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_subsector written to all regions") %>%
+      add_legacy_name("L202.SubsectorAll_in") %>%
+      add_precursors("aglu/A_an_input_subsector", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.SubsectorAll_in
+
+    L202.StubTech_in %>%
+      add_title("Identification of stub technologies for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology written to all regions") %>%
+      add_legacy_name("L202.StubTech_in") %>%
+      add_precursors("aglu/A_an_input_technology", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.StubTech_in
+
+    L202.StubTechInterp_in %>%
+      add_title("Generic technology info for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology written to all regions") %>%
+      add_legacy_name("L202.StubTechInterp_in") %>%
+      add_precursors("aglu/A_an_input_technology", "energy/A_regions", "common/GCAM_region_names") ->
+      L202.StubTechInterp_in
+
+    L202.GlobalTechCoef_in %>%
+      add_title("Coefficients for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_technology across all base and future years") %>%
+      add_legacy_name("L202.GlobalTechCoef_in") %>%
+      add_precursors("aglu/A_an_input_technology", "common/GCAM_region_names") ->
+      L202.GlobalTechCoef_in
+
+    L202.GlobalTechShrwt_in %>%
+      add_title("Default shareweights for inputs to animal production") %>%
+      add_units("NA") %>%
+      add_comments("A_an_input_globaltech_shrwt interpolated to all model years") %>%
+      add_legacy_name("L202.GlobalTechShrwt_in") %>%
+      add_precursors("aglu/A_an_input_globaltech_shrwt", "common/GCAM_region_names") ->
+      L202.GlobalTechShrwt_in
+
+    L202.StubTechProd_in %>%
+      add_title("Base year output of the inputs (feed types) to animal production") %>%
+      add_units("units") %>%   # TODO - what is calOutputValue units here?
```

Comment:
Mt/yr
For comments: Calibrated primary sources of animal feed commodities, specific to each region and time period.